src/react-native/react-native-wizard.ts

```diff
@@ -0,0 +1,256 @@
+/* eslint-disable max-lines */
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as process from 'process';
+import {
+  addSentryCliConfig,
+  confirmContinueIfNoOrDirtyGitRepo,
+  confirmContinueIfPackageVersionNotSupported,
+  ensurePackageIsInstalled,
+  getOrAskForProjectData,
+  getPackageDotJson,
+  installPackage,
+  printWelcome,
+  propertiesCliSetupConfig,
+} from '../utils/clack-utils';
+import { getPackageVersion, hasPackageInstalled } from '../utils/package-json';
+import { podInstall } from '../apple/cocoapod';
+import { platform } from 'os';
+import {
+  getValidExistingBuildPhases,
+  findBundlePhase,
+  patchBundlePhase,
+  findDebugFilesUploadPhase,
+  addDebugFilesUploadPhase,
+  writeXcodeProject,
+} from './xcode';
+import {
+  doesAppBuildGradleIncludeRNSentryGradlePlugin,
+  addRNSentryGradlePlugin,
+  writeAppBuildGradle,
+} from './gradle';
+import { runReactNativeUninstall } from './uninstall';
+import { APP_BUILD_GRADLE, XCODE_PROJECT, getFirstMatchedPath } from './glob';
+import { ReactNativeWizardOptions } from './options';
+import { SentryProjectData } from '../utils/types';
+import {
+  addSentryInitWithSdkImport,
+  doesJsCodeIncludeSdkSentryImport,
+} from './javascript';
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+const xcode = require('xcode');
+
+export const RN_SDK_PACKAGE = '@sentry/react-native';
+
+export const RN_PACKAGE = 'react-native';
+export const RN_HUMAN_NAME = 'React Native';
+
+export const SUPPORTED_RN_RANGE = '>=0.69.0';
+
+export async function runReactNativeWizard(
+  options: ReactNativeWizardOptions,
+): Promise<void> {
+  options.telemetryEnabled = false;
+
+  if (options.uninstall) {
+    return runReactNativeUninstall(options);
+  }
```

Comment:
Ahh this is the first clack-based wizard with an uninstall functionality. I like the simplicity of this approach!

Reply:
Thanks.